test/Microsoft.Health.Fhir.Shared.Tests.Integration/Persistence/SqlQueueClientTests.cs

```diff
@@ -259,6 +260,40 @@ public async Task GivenGroupJobs_WhenCancelJobsByGroupId_ThenAllJobsShouldBeCanc
             Assert.Equal(jobInfo2.Result, jobInfo.Result);
         }
 
+        [Fact]
+        public async Task GivenGroupJobs_WhenCancelJobsByGroupIdCalledTwoTimes_ThenJobStatusShouldNotChange()
```

Comment:
>TwoTimes [](http://example.com/codeflow?start=70&length=8)

Twice

Reply:
Sorry, I will modify the name of the function.